src/main/java/com/autotune/database/service/ExperimentDBService.java

```diff
@@ -94,27 +95,38 @@ public boolean getRecommendationToSave(Map<String, KruizeObject> experimentsMap,
      * The method then converts the retrieved data into KruizeObject format, adds them to a list,
      * and sends it to the ExperimentInterface implementation to store the objects.
      */
-    public void loadAllExperiments() throws Exception {
+    public void loadAllExperimentsData() throws Exception {
         List<KruizeExperimentEntry> entries = experimentDAO.loadAllExperiments();
         List<CreateExperimentAPIObject> createExperimentAPIObjects = DBHelpers.Converters.KruizeObjectConverters.convertExperimentEntryToCreateExperimentAPIObject(entries);
         List<KruizeObject> kruizeExpList = new ArrayList<>();
+        List<KruizeResultsEntry> kruizeResultsEntries = experimentDAO.loadAllResults();
+        List<UpdateResultsAPIObject> updateResultsAPIObjects = DBHelpers.Converters.KruizeObjectConverters.convertResultEntryToUpdateResultsAPIObject(kruizeResultsEntries);
+        List<ExperimentResultData> resultDataList = new ArrayList<>();
```

Comment:
move this after throw caluse

Reply:
if (failureThreshHold > 0 && failureCount == failureThreshHold) {
            throw new Exception("None of the experiments are able to load from DB.");
            throw new Exception("None of the experiments are able to load from DB.");
        }
        }

Reply:
Done